Formula/rpm.rb

```diff
@@ -34,6 +34,13 @@ class Rpm < Formula
   depends_on "gettext"
   depends_on "xz"
   depends_on "ossp-uuid"
+  # use external libraries, until the internal have been fixed
+  depends_on "lua"
+  depends_on "syck"
+  depends_on "openssl"
+
+  # lots of OS X support things were missing from this release
+  patch :DATA
```

Comment:
Well yes, I will likely have a rpm-5.4.17 tarball in less than a week. Meanwhile, there are reasons to patch the tarball in order to 1) change configuration 2) change the build. E.g. there are no important changes to syck or lua (nor are they critical to using rpm on any platform). Note also that I target the latest released Berkeley DB (now 6.2.23) and ECDSA/SHA256 (in libtomcrypt by default, but also present in NSS/OpenSSL/libgcrypt, but not present in BeeCrypt). I expect that distress/projects change how rpm is built by applying patches to the rpm-5.4.17 tarsal. So I'm a but surprised that you are waiting for an unpatched tarball from "upstream".